packages/expo-image/android/src/main/java/expo/modules/image/okhttp/GlideUrlWrapperLoader.kt

```diff
@@ -28,7 +28,7 @@ class GlideUrlWrapperLoader(
             originalResponse
               .newBuilder()
               .body(
-                ProgressResponseBody(originalResponse.body) { bytesWritten, contentLength, done ->
+                ProgressResponseBody(originalResponse.body ?: throw NullPointerException()) { bytesWritten, contentLength, done ->
```

Comment:
```suggestion
                ProgressResponseBody(requireNotNull(originalResponse.body)) { bytesWritten, contentLength, done ->
```

nit: NullPointerException is not quite precise and we can just use requireNotNull